account/views.py

```diff
@@ -0,0 +1,56 @@
+import json, jwt, bcrypt
+from django.views   import View
+from django.http    import JsonResponse, HttpResponse
+from .models        import Account
+from adidas.settings import SECRET_KEY
+
+class SignInView(View):
+    def post(self, request):
+        account_data = json.loads(request.body)
+
+        try:
+            print('try in')
+            if Account.objects.filter(user_account=account_data['user_account']).exists():
+                account = Account.objects.get(user_account=account_data['user_account'])
+                print('if one in')
+                if bcrypt.checkpw(account_data['password'].encode('utf-8'), account.password.encode('utf-8')):
+                    token = jwt.encode({'user_account' : account.id }, SECRET_KEY, algorithm = 'HS256')
+                    return JsonResponse({ 'access_token' : token.decode('utf-8')}, status=200)
+
+                return JsonResponse({ 'message' : 'Error' }, status=401)
```

Comment:
패스워드가 틀리면 나는 에러인데 보통이런경우 자세히 알려주지 않는게 보안상 좋기 때문에 HttpResponse(status=401) 처리하는게 더 깔끔합니다.
다만, 현재 1차프로젝트이고, 프론트에서 이부분에 대해서 불명확해 한다면 
```suggestion
                return JsonResponse({ 'message' : 'INVALID_PASSWORD' }, status=401)
```
와 같이 명확히 명시하는게 더 낫습니다.